server/core/SqlQueryBuilder.js

```diff
@@ -356,6 +362,14 @@ class SqlQueryBuilder {
             sql = `${field} IS NOT NULL`;
             break;
 
+        case 'empty':
+            sql = `${field} = ' '`;
```

Comment:
Can a date field be empty ?
The previous comment about TRIM() might not be relevant for date fields

Reply:
It's not a good idea to compare date with this type of filter